tinynumpy/tinynumpy.py

```diff
@@ -408,7 +408,13 @@ def cross(u, v):
     uxv = []
 
     # http://mathworld.wolfram.com/CrossProduct.html
-    if uDim == vDim == 3:
+    if uDim == vDim == 2:
+        try:
+            uxv.append(0)
```

Comment:
Hi there, can you explain the purpose of appending this 0? It doesn't affect the result.